app/src/main/java/com/mk/tasky/authentication/data/AuthenticationRepositoryImpl.kt

```diff
@@ -28,7 +29,25 @@ class AuthenticationRepositoryImpl(
         }
     }
 
-    private fun parseError(e: HttpException): Result<LoggedUser> {
+    override suspend fun register(
+        name: String,
+        email: String,
+        password: String
+    ): Result<Boolean> {
+        val body = RegistrationBodyDto(fullName = name, email = email, password = password)
+        return try {
+            api.register(body)
+            Result.success(true)
```

Comment:
There wouldn't really be a case where you'd return a Result.success(false), right? In these cases, I personally make it a `Result<Nothing>` or `Result<Unit>` since whether the registration was successful is already decided by the Result itself.

Reply:
Fixed!